specs/avatar.md

```diff
@@ -0,0 +1,105 @@
+# Avatar
+
+## Overview
+
+An avatar is a graphical represention of a user or object.
+
+### Use Cases
+
+A common use case would be to display an image or text (usually initials) of a user or an object.
+
+### Features
+- Image slot: Able to slot in an avatar image
+- Badge slot: Able to slot in a badge component
+- `size`, a numeric value that determines the size of the avatar
+- `shape`, a circle or square shape can be chosen
+- `color`, a hexadecimal color can be provided to determine the coin background color
+- If no image is provided, the initials of the provided name will be displayed
+
+### Prior Art/Examples
+
+- [Fluent UI (Persona)](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)
+- [Fluent UI](https://fluentsite.z22.web.core.windows.net/components/avatar/definition)
+- [Lighting Design](https://www.lightningdesignsystem.com/components/avatar/)
+- [Evergreen](https://evergreen.segment.com/components/avatar)
+- [Ant Design](https://ant.design/components/avatar/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/design-system/avatar)
+---
+### API
+
+*Component Name*
+- `fast-avatar`
+
+#### Attributes
+|   Name    | Description                                  | Type                |
+|-----------|----------------------------------------------|---------------------|
+| `name`    | Accepts name string for coin display         | `string`            |
+| `shape`   | Determines the avatar coin shape.            | `circle | square`   |
+| `fill`    | Sets the background fill of the avatar coin. | `string: hex color` |
+| `color`   | Sets the color of the avatar coin text.      | `string: hex color` |
+| `size`    | Sets the size of avatar coin.                | `number`            |
+
+### Anatomy and Appearance
+
+*Template*
+```
+<div class="coin">
+  <a href="${ x => x.link}">
+    <span class="name">
+      ${x => x.name}
+    </span>
+    ${when(x => x.imgSrc, html<TestAvatar>`
+      <img class="photo"
+        alt="${ x => x.altText }" 
+        src="${ x => x.imgSrc }" />
+    `)}
+
+  </a>
+</div>
+<slot class="badge" name="badge"></slot>
+```
+
+---
+
+## Implementation
+
+```
+<test-avatar 
+  imgSrc=""
+  altText="Jenny's profile image"
```

Comment:
HTML does not have capital letters, these should be `img-src` and `alt-text`. I would also think the name of this would be `fast-avatar`. You also are specifying above that there is an image slot, yet here you create an image and apply some attributes to it that have been passed. Which are you intending to do?

Reply:
Thanks for your feedback! I'll revise the attribute names. The intent is to have an image attribute. It seems I left an old idea in there. Will revise and remove.

Reply:
This has been addressed.